src/components/layout/footer.tsx

```
@@ -0,0 +1,74 @@
+
+type Commit = {
+    sha: string
+    commit: {
+        message: string
+        author: {
+            date: string
+        }
+    }
+}
+
+const GITHUB_API_URL = 'https://api.github.com/repos/remcostoeten/ripgrep-command-creator-helper/commits'
```

ESLint

[error] 'GITHUB_API_URL' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)